lambda/decorate_prowler_findings.py

```diff
@@ -125,19 +127,42 @@ def process_account_findings(findings_to_process: List[Dict]) -> List[Dict]:
             ddb_finding.pop("unmapped", None)
             processed_findings.append(f)
             # Don't want to add these to the output file (unnecessary), but do want them to be added to DDB
-            ddb_finding["finding_info_uid"] = ddb_finding["finding_info"]["uid"]
-            ddb_finding["metadata_event_code"] = ddb_finding["metadata"]["event_code"]
-            ddb_finding["cloud_account_uid"] = ddb_finding["cloud"]["account"]["uid"]
-            findings_to_add.append(ddb_finding)
+            ddb_finding["finding_info_uid"] = ddb_finding.get("finding_info", {}).get("uid")
```

Comment:
I think we need to remove these. There is probably the need to catch these elsewhere.